### ISSUE TITLE
no validation loss

### ISSUE THREAD

**Issue by daveredrum** [yunjey/show-attend-and-tell-tensorflow#68]
I'm pretty confused because there's no loss calculated in validation phase. Normally there should be some validation loss/accuracy/perplexity, especially for CV tasks. However there's only some NLP metrics for captions generated in validation phase.

Do we really need to calculate validation loss for the generated captions?

**Comment by zBabar** [comment]
Hi... have implemented and computed validation loss??? Does it work?
